openupgradelib/openupgrade.py

```diff
@@ -969,7 +1081,7 @@ def write_value(ids, field, value):
                 write_value(ids, field, value)
 
 
-def logged_query(cr, query, args=None, skip_no_result=False):
+def logged_query(cr, query, args=(), skip_no_result=False):
```

Comment:
Don't change `args=None` to `args=()` or you will have the problem with mutable variables:

http://effbot.org/zone/default-values.htm

Reply:
But tuples are immutable... :thinking:

Reply:
And then any overwriting without check will do this to fail while trying to append. This expects to be `None` or a list, not a tuple. Why don't you simply keep as it was?